HPXMLtoOpenStudio/resources/hpxml.rb

```diff
@@ -1984,8 +1986,9 @@ def from_oga(hpxml)
 
   class Foundation < BaseElement
     ATTRS = [:id, :foundation_type, :vented_crawlspace_sla, :within_infiltration_volume,
-             :attached_to_slab_idrefs, :attached_to_floor_idrefs, :attached_to_foundation_wall_idrefs,
-             :attached_to_wall_idrefs, :attached_to_rim_joist_idrefs]
+             :skirt_present, :attached_to_slab_idrefs, :attached_to_floor_idrefs,
```

Comment:
Should probably call this `belly_wing_skirt_present` or similar, to clarify that it only applies to that foundation type.